CloneInstagram/CloneInstagram/Controllers/ProfileDetailViewController.swift

```diff
@@ -0,0 +1,117 @@
+//
+//  ProfileDetailViewController.swift
+//  CloneInstagram
+//
+//  Created by Oleg_Yakovlev on 22.10.22.
+//
+
+import UIKit
+
+// Экран детальной информации
+final class ProfileDetailViewController: UIViewController {
+    private enum Constants {
+        static let identifiers: [Identifiers] = [.profileSubscribeLikes, .profileInfoDetail, .actuality, .postInfo]
+    }
+    
+    private enum Identifiers: String {
```

Comment:
```suggestion
  Identifiers
```
тут поменяй везде окончания как следует